### ISSUE TITLE
[Feature Request] [integration] tree-sitter-markdown

### ISSUE THREAD

**Issue by B4rc1** [EdenEast/nightfox.nvim#61]
After seeing this [reddit post](https://www.reddit.com/r/neovim/comments/rg97j4/treesitter_for_markdown/)  and this [comment](https://www.reddit.com/r/neovim/comments/rg97j4/comment/hoje1mg/?utm_source=share&utm_medium=web2x&context=3) I would love to see support for tree-sitter-markdown.

**Comment by EdenEast** [comment]
The markdown parser is setting highlight groups that are already defined. Nightfox sets some of these groups. Others for example will fallback to the non typescript variant.  For example: nightfox does not define `TSTitle` This means that it falls back to the normal `Title` group definition. That is why titles are blue even though nightfox does not define `TSTitle`.

I recently added `TSURI` to help make urls more visible in a markdown file with this tree-sitter parser.

To summarize nightfox already supports this as it uses already defined groups.
